### PR TITLE
3A 

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: walkthroughs/3a-low-code-api-development/walkthrough.adoc [hunk 1]
```diff
@@ -10,22 +10,20 @@
...
+In this walkthrough you follow a design-first approach, based on the OpenAPI specification, to define a RESTful API that accepts a name parameter and echoes it back in the response body. 
+You then use the resulting Open API definition to generate the code for your API and deploy it in a containerised environment.
 
-Following a design first approach to API development is beneficial since it is self-documenting, and provides a clear API definition for partners and end-users of your API, and allows you to identify important design decisions prior coding and deployment.
-
-To create this integration, you will use Integration capabilities in link:{fuse-url}[Red Hat Fuse Online, window="_blank"].
-
-All of the applications and services used in the walkthrough are running on Red Hat OpenShift.
+Following a design first approach to API development is self-documenting, and provides a clear API definition for partners and end-users of your API.
+It also allows you to identify important design decisions prior coding and deployment.
```
Comment (evanshortiss) [inline walkthroughs/3a-low-code-api-development/walkthrough.adoc]: "prior to"

File: walkthroughs/3a-low-code-api-development/walkthrough.adoc [hunk 2]
```diff
@@ -42,49 +40,61 @@ All of the applications and services used in the walkthrough are running on Red
...
+. Click *Skip for now* when prompted to add team members.
 
 // verify that you can access https://your-worksapce-name.slack.com.
 [type=verification]
-Can you access your workspace at *your-workspace-name.slack.com*, and see the *#slack-demo* channel listed on the left of the workspace?
+Can you access your workspace at `https://<your-workspace-name>.slack.com`, and navigate to the *slack-demo* channel listed on the left of the workspace?
+
 
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact Slack.
```
Comment (evanshortiss) [inline walkthroughs/3a-low-code-api-development/walkthrough.adoc]: "slack support"?

File: walkthroughs/3a-low-code-api-development/walkthrough.adoc [hunk 3]
```diff
@@ -151,7 +169,7 @@ A *Path* is used to represent an API endpoint/operation and the associated param
 
 . Click your new */greeting* endpoint under the *Paths* heading.
 
-. Click *Create Operation* under the *POST* icon in the *Operations* section on the right. The *POST* icon will become orange.
+. Click *Create Operation* under the *POST* icon in the *Operations* section on the right. The *POST* icon become orange.
```
Comment (evanshortiss) [inline walkthroughs/3a-low-code-api-development/walkthrough.adoc]: "become" => "becomes" or "will become"?

File: walkthroughs/3a-low-code-api-development/walkthrough.adoc [hunk 4]
```diff
@@ -235,8 +262,18 @@ By default OpenShift does not create a public endpoint, known as a *Route*, that
...
-=== Invoke the API
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
+=== Invoking the API
+
+. Use an HTTP client to invoke the route.
+
+[type=verification]
+Did the message "Hello from $YOUR_NAME" appears in your Slack channel?
```
Comment (evanshortiss) [inline walkthroughs/3a-low-code-api-development/walkthrough.adoc]: "appears" => "appear"

**Review by evanshortiss — COMMENTED** [review]
Looks good @finp, just some typos from what I can see. The changes I'll add once this is merged are straightforward so it should be quick 👍

File: walkthroughs/3a-low-code-api-development/walkthrough.adoc [hunk 5]
```diff
@@ -42,49 +40,61 @@ All of the applications and services used in the walkthrough are running on Red
...
-. Enter a name for your workspace. The name you choose will ultimiately become part of the URL. For example, naming the workspace fuseonline would produce a URL of fuseonlinedemo.slack.com. Click *Next*.
+. Enter your email address and click *Next*.
 
-. When asked *What is your team working on?* enter *Slack Demo*. This will create a *#slack-demo* channel we can use.
+. Enter a name for your workspace and click *Next*.
+The name you choose becomes part of the URL for the workspace. 
+For example, naming the workspace `fuseonline` creates the URL `fuseonlinedemo.slack.com`.
 
-. Click *Skip for now* when prompted to add team members.
+. When promted *What is your team working on?* enter `Slack Demo`.
```
Comment (tremes) [inline walkthroughs/3a-low-code-api-development/walkthrough.adoc]: Typo `prompted`.

File: walkthroughs/3a-low-code-api-development/walkthrough.adoc [hunk 6]
```diff
@@ -181,8 +194,16 @@ Is a *Greeting Slack Target* entry is listed in the Fuse Online *Connections* sc
 
 . Click *Save and Continue* to save your API design and move onto implementation.
 
+
+[type=verification]
+Is a *Greeting Slack Target* entry is listed in the Fuse Online *Connections* screen?
```
Comment (tremes) [inline walkthroughs/3a-low-code-api-development/walkthrough.adoc:199]: Double is.

**Comment by pwright** [comment]
merging now to allow Evan perform updates. Also need to follow up with more edits before eventual merge to master

**Pull request merged**